commit 6707e6f7e0661b0acb5ba78789d66cdc5b75dff6
Author: Aws Ismail <aws.ismail@windriver.com>
Date:   Thu May 31 18:48:51 2012 -0400

    This patch was taken initially from OE:
    
    http://git.openembedded.org/openembedded/commit/recipes/corosync/files/corosync-docs.patch?id=5486dcbf08200c22b6086d37055fb64de423eabe
    
    It had the following log:
    
        Update to 1.2.8
        Add corosync-docs.patch to fix html doc compilation
    
        Signed-off-by: Khem Raj <raj.khem@gmail.com>
    
    I have decided to update this patch to two related patches and update them
    for corosync v.1.4.3. This patch is now:
    1. corosync-docs-in-configure-ac.patch
    2. corosync-docs-in-makefile-am.patch
    
    This patch is also related to corosync-wr-no-html-docs.patch from WRL4.3
    which was dropped since this updated patch allows the build of docs
    to be optional.
    
    Upstream Status: Pending
    
    Signed-off-by: Aws Ismail <aws.ismail@windriver.com>

diff --git a/configure.ac b/configure.ac
index d78c549..728c51a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -554,7 +554,7 @@ if test "x${enable_user_flags}" = xyes; then
 fi
 
 # final build of *FLAGS
-CFLAGS="$ENV_CFLAGS $OPT_CFLAGS $GDB_FLAGS $OS_CFLAGS \
+CFLAGS="$ENV_CFLAGS $OS_CFLAGS \
 	$COVERAGE_CFLAGS $EXTRA_WARNINGS $WERROR_CFLAGS $NSS_CFLAGS"
 CPPFLAGS="$ENV_CPPFLAGS $ANSI_CPPFLAGS $OS_CPPFLAGS"
 LDFLAGS="$ENV_LDFLAGS $COVERAGE_LDFLAGS $OS_LDFLAGS"

diff --git a/configure.ac b/configure.ac
index 728c51a..38b31e4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -581,7 +581,10 @@ AM_CONDITIONAL(BUILD_SOLARIS, test -n "${SOLARIS_OPTS}")
 AC_SUBST([DARWIN_OPTS])
 AC_SUBST([SOLARIS_OPTS])
 
-AM_CONDITIONAL(BUILD_HTML_DOCS, test -n "${GROFF}")
+AC_ARG_ENABLE([doc],
+   AS_HELP_STRING([--enable-doc],[Build html documentation])
+)
+AM_CONDITIONAL(BUILD_HTML_DOCS, test -n "${GROFF}" && test "x$enable_doc" = "xyes")
 
 AC_SUBST([LINT_FLAGS])
